### PR TITLE
consensus/docs: latest roadmap of remaining Genesis work

### PR DESCRIPTION
**Pull request by nfrisby**
Sketching another roadmap.

### CODE REVIEW

**Comment by nfrisby** [comment]
You can see this file rendered nicely here https://github.com/input-output-hk/ouroboros-network/blob/nfrisby/2023-jan-genesis-roadmap/ouroboros-consensus/docs/2023-Jan-Genesis-roadmap.md

File: ouroboros-consensus/docs/2023-Jan-Genesis-roadmap.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    ProtoDR2["update the DR prototype"]
+    ProtoCSJ2["update the CSJ prototype,\nnow incl. slow path"]
+    ProtoLoP2["update the CSC prototype"]
+
+    ProtoCSJ2 --> DocCSJ2["document updated DR\n(incl. justification)"]
+    ProtoDR2  --> DocDR2["document updated CSJ\n(incl. justification)"]
+    ProtoLoP2 --> DocLoP2["document updated CSC\n(incl. justification)"]
+
+    DocDR2  --> TestDR2{test DR}
+    DocCSJ2 --> TestCSJ2{test CSJ (incl. both fast and slow)}
```
Comment (amesgen) [inline ouroboros-consensus/docs/2023-Jan-Genesis-roadmap.md:82]: This diagram does not render, this should fix it:
```suggestion
    DocCSJ2 --> TestCSJ2{"test CSJ (incl. both fast and slow)"}
```

File: ouroboros-consensus/docs/2023-Jan-Genesis-roadmap.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    DocCSJ --> TestCSJ{smoke test fast CSJ}
+    DocLoP --> TestLoP{smoke test CSC}
+    style TestDR fill:#c27832
+    style TestCSJ fill:#c27832
+    style TestLoP fill:#c27832
+```
+
+## 2nd Engineering Phase
+
+We expect the first review to yeild some alterations, which we now incorporate into the prototypes and the tests
```
Comment (amesgen) [inline ouroboros-consensus/docs/2023-Jan-Genesis-roadmap.md:69]:
```suggestion
We expect the first review to yield some alterations, which we now incorporate into the prototypes and the tests
```

**Review by amesgen — APPROVED** [review]

File: ouroboros-consensus/docs/2023-Jan-Genesis-roadmap.md (hunk 1 above)
Comment (nfrisby) [reply ouroboros-consensus/docs/2023-Jan-Genesis-roadmap.md]: Fixed

**Comment by nfrisby** [comment]
bors merge

**Pull request merged**